internal/service/lexv2models/slot.go

```diff
@@ -0,0 +1,894 @@
+	MaxRetries                  types.Int64                                                    `tfsdk:"max_retries"`
+	MessageGroup                fwtypes.ListNestedObjectValueOf[MessageGroupData]              `tfsdk:"message_groups"`
+	MessageSelectionStrategy    fwtypes.StringEnum[awstypes.MessageSelectionStrategy]          `tfsdk:"message_selection_strategy"`
+	PromptAttemptsSpecification fwtypes.SetNestedObjectValueOf[PromptAttemptSpecificationData] `tfsdk:"prompt_attempts_specification"`
+}
+type PromptAttemptSpecificationData struct {
+	AllowedInputTypes              fwtypes.ListNestedObjectValueOf[AllowedInputTypesData]              `tfsdk:"allowed_input_types"`
+	AllowInterrupt                 types.Bool                                                          `tfsdk:"allow_interrupt"`
+	MapBlockKey                    fwtypes.StringEnum[PromptAttemptsType]                              `tfsdk:"map_block_key"`
+	AudioAndDTMFInputSpecification fwtypes.ListNestedObjectValueOf[AudioAndDTMFInputSpecificationData] `tfsdk:"audio_and_dtmf_input_specification"`
+	TextInputSpecification         fwtypes.ListNestedObjectValueOf[TextInputSpecificationData]         `tfsdk:"text_input_specification"`
+}
+
+type DTMFSpecificationData struct {
+	EndCharacter      types.String `tfsdk:"end_character"`
+	EndTimeoutMs      types.Int64  `tfsdk:"end_timeout_ms"`
+	DeletionCharacter types.String `tfsdk:"deletion_character"`
+	MaxLength         types.Int64  `tfsdk:"max_length"`
+}
+
+type TextInputSpecificationData struct {
+	StartTimeoutMs types.Int64 `tfsdk:"start_timeout_ms"`
+}
+
+type AllowedInputTypesData struct {
+	AllowAudioInput types.Bool `tfsdk:"allow_audio_input"`
+	AllowDTMFInput  types.Bool `tfsdk:"allow_dtmf_input"`
+}
+
+type AudioAndDTMFInputSpecificationData struct {
+	AudioSpecification fwtypes.ListNestedObjectValueOf[AudioSpecificationData] `tfsdk:"audio_specification"`
+	StartTimeoutMs     types.Int64                                             `tfsdk:"start_timeout_ms"`
+	DTMFSpecification  fwtypes.ListNestedObjectValueOf[DTMFSpecificationData]  `tfsdk:"dtmf_specification"`
+}
+
+type AudioSpecificationData struct {
+	EndTimeoutMs types.Int64 `tfsdk:"end_timeout_ms"`
+	MaxLengthMs  types.Int64 `tfsdk:"max_length_ms"`
+}
+
+type CustomPayloadData struct {
+	Value types.String `tfsdk:"value"`
+}
+
+type ImageResponseCardData struct {
+	Title    types.String                                `tfsdk:"title"`
+	Button   fwtypes.ListNestedObjectValueOf[ButtonData] `tfsdk:"buttons"`
+	ImageURL types.String                                `tfsdk:"image_url"`
+	Subtitle types.String                                `tfsdk:"subtitle"`
+}
+
+type ButtonData struct {
+	Text  types.String `tfsdk:"text"`
+	Value types.String `tfsdk:"value"`
+}
+
+type PlainTextMessageData struct {
+	Value types.String `tfsdk:"value"`
+}
+
+type SSMLMessageData struct {
+	Value types.String `tfsdk:"value"`
+}
+type MessageGroupData struct {
```

Comment:
Same. Here you are using "Variations" but the correct is "Variation" since there will be a block for each and it doesn't make as much sense for each block to be plural. Autoflex handles this but you might need to change test cases.